samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/publisher.c

```diff
@@ -241,4 +241,3 @@ void publish(struct k_work *work)
 		printk("bt_mesh_model_publish: err: %d\n", err);
 	}
 }
-
```

Comment:
How is this change related to the commit?

Reply:
done !!